src/components/AssignmentTexter/Controls.jsx

```diff
@@ -171,6 +171,12 @@ export class AssignmentTexterContactControls extends React.Component {
   };
 
   onKeyUp = evt => {
+    if (evt.target && evt.target.id === "contact-notes-input") {
```

Comment:
oh this is a good catch and is nice and neat. One follow up here: is this the only texter sidebox that should overwrite this behavior? One thought would be to do:

```suggestion
    const texterSideboxesThatAllowInput = ["contact-notes-input"]
    if (evt.target && texterSideboxesThatAllowInput.includes(evt.target.id)) {
```

Or something similar if we think we'd want this to be reused. Otherwise, this is good!

Reply:
excellent suggestion, implemented with a slight tweak :)
https://github.com/MoveOnOrg/Spoke/pull/2192/commits/257400e30a26544526f97cc4a454cf38b24eb162